app/src/main/java/com/example/lhi06/residue/AlbumAdapter.java

```diff
@@ -77,9 +77,9 @@ public int getItemCount() {
     }
 
     public void removeAt(@NonNull final int position) {
-        albumList.remove(position);
+        albums.remove(position);
         notifyItemRemoved(position);
-        notifyItemRangeChanged(0, albumList.size());
+        notifyItemRangeChanged(0, albums.size());
```

Comment:
i havent actually used this method it was just something that a tutorial i was following said to add, so im not quite sure what the second line does